lang/py/avro/__init__.py

```diff
@@ -19,4 +19,14 @@
 
 from __future__ import absolute_import, division, print_function
 
+import os
+
 __all__ = ['schema', 'io', 'datafile', 'protocol', 'ipc', 'constants', 'timezones', 'codecs']
+
+def LoadResource(name):
+  dir_path = os.path.dirname(__file__)
+  rsrc_path = os.path.join(dir_path, name)
+  with open(rsrc_path, 'r') as f:
+    return f.read()
+
+__version__ = LoadResource('VERSION.txt').strip()
```

Comment:
Consider using [`pkgutil.get_data`](https://docs.python.org/3/library/pkgutil.html) here:

```
import pkgutil

__version__ = (pkgutil.get_data(__name__, 'VERSION.txt') or b'0.0.1+unknown').decode().strip()
```

pkgutil.get_data pretty much does what your LoadResource does, but it's zip-safe. On the flip side, it returns `Optional[bytes]`, so we need a little code to ensure there's a usable value if VERSION.txt is not found.

Reply:
Thanks for the review! I refactored the code with your suggestd code.